src/main/resources/org/sagebionetworks/web/client/widget/entity/menu/v2/ActionMenuWidgetViewImpl.ui.xml

```diff
@@ -8,7 +8,7 @@
 	xmlns:w="urn:import:org.sagebionetworks.web.client.widget">
 
 	<g:FlowPanel>
-		<b:Tooltip title="Show advanced search">
+		<b:Tooltip title="Edit view query">
```

Comment:
```suggestion
		<b:Tooltip title="Edit query">
```

This wasn't obvious from the design, but this tooltip needs to be dynamic or generic enough for all tables (not just Views, which are a type of table). Since this piece is built into the Explore component that's being ported, I'd say it's not worth the effort to make it dynamic